commitlint.config.ts

```diff
@@ -57,11 +60,35 @@ module.exports = {
         // * Workflow: detect if wip commit in a branch not named "wip/*" or whose name contains "squashed".
         // * Allow PascalCase word after colon in title (exception to subject-lowercase rule), e.g.: "End2End: TestFixtureSetup refactor"
         // * Detect if commit hash mention in commit msg actually exists in repo.
-        // * Give replacement suggestions in rule that detects too long titles (e.g. and->&, config->cfg, ...)
         // * Detect area(sub-area) in the title that doesn't include area part (e.g., writing (bar) instead of foo(bar))
 
         {
             rules: {
+                'header-max-length-with-suggestions': ({header}: {header:any}, _: any, headerMaxLineLength:any) => {
```

Comment:
@realmarv let's rename this parameter to just maxLineLength (it can be deduced that it is referring to the header, just by looking at the rule name); this way you can uncomment the global var headerMaxLineLength at line 19 and use it at line 30

Reply:
Done